src/components/Register.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import '../assets/styles/register.scss'
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import UserService from '../services/UserService';
+
+const SignupSchema = Yup.object().shape({
+    first_name: Yup.string()
+        .min(2, 'Too Short!')
+        .max(50, 'Too Long!')
+        .required('Required'),
+    last_name: Yup.string()
+        .min(2, 'Too Short!')
+        .max(50, 'Too Long!')
+        .required('Required'),
+    email: Yup.string().email('Invalid email').required('Required'),
+    password: Yup.string().min(6, 'You must enter 6 characters at least!').required('Password is required')
+});
+
+const Register = ({history}) => {
+    return (
+        <div>
+            <Formik
+                initialValues={{ email: '', password: '', first_name: '', last_name: '' }}
+                validate={values => {
+                    const errors = {};
+
+                    if (!values.email) {
```

Comment:
Do we need these validations or yup schema is already handling them (with `required()`)?
Also, I'm not sure that these conditions will ever evaluate to `false`, as the default values for these fields are empty strings and expressions like `!''` or `!'<any string here>` evaluate to `true`.